packages/manager/apps/veeam-backup/src/pages/listing/Listing.page.tsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { OsdsButton, OsdsLink } from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  Datagrid,
+  DataGridTextCell,
+  BaseLayout,
+  DatagridColumn,
+  ActionMenu,
+  useDataGrid,
+} from '@ovhcloud/manager-components';
+import Loading from '@/components/Loading/Loading';
+import ErrorBanner from '@/components/Error/Error';
+import { Breadcrumb } from '@/components/Breadcrumb/Breadcrumb';
+import { urls } from '@/routes/routes.constant';
+import { SuccessMessages } from '@/components/Messages/SuccessMessage.component';
+import {
+  VeeamBackupWithIam,
+  getDisplayName,
+  useOrganizations,
+  useRegions,
+  useVeeamBackupList,
+  getVeeamBackupVCDOrganizationDisplayName,
+} from '@/data';
+import { StatusCell } from './StatusCell.component';
+import { CreatedAtCell } from './CreatedAtCell.component';
+import { sortVeeamBackups } from './sorting';
+
+export default function Listing() {
+  const { t } = useTranslation('listing');
+  const [flattenData, setFlattenData] = React.useState([]);
+  const navigate = useNavigate();
+  const { sorting, setSorting } = useDataGrid({ id: 'name', desc: false });
+  const {
+    data,
+    fetchNextPage,
+    hasNextPage,
+    isError,
+    isLoading,
+    error,
+    status,
+  } = useVeeamBackupList({ pageSize: 10 });
+  const { data: regions } = useRegions();
+  const { data: organizations } = useOrganizations();
```

Comment:
**nitpick:** You can directly use `useIcebergV2` hook from `manager-components` to fetch the data.

Reply:
done fixed